packages/cli/src/commands/mock.ts

```diff
@@ -46,15 +53,15 @@ const mockCommand: CommandModule = {
         },
       }),
   handler: parsedArgs => {
-    const { multiprocess, dynamic, port, host, spec } = (parsedArgs as unknown) as CreatePrismOptions & {
+    const { multiprocess, dynamic, port, host, operations } = (parsedArgs as unknown) as CreatePrismOptions & {
       multiprocess: boolean;
     };
 
     if (multiprocess) {
-      return createMultiProcessPrism({ dynamic, port, host, spec });
+      return (parsedArgs.p = createMultiProcessPrism({ dynamic, port, host, operations }));
```

Comment:
# Yes
## You're right!
### This is exactly what you think it is!

This is yet another test that has been passing forever incorrectly and I discovered it because I've removed 30192930189031283 layers that Prism was not using. We need a reference here to correctly wait the command to be processed before start doing assertions